library/std/src/panic.rs

```diff
@@ -314,6 +314,13 @@ impl<T> DerefMut for AssertUnwindSafe<T> {
     }
 }
 
+#[stable(feature = "catch_unwind", since = "1.49.0")]
```

Comment:
This should be something like `#[stable(feature = "assert_unwind_safe_clone", since = "1.51.0")]`

Reply:
Fixed!